sort.py

```diff
@@ -0,0 +1,22 @@
+
+def sort(array):
+    if not array or len(array) <= 1:
+        return array # nothing to sort
+
+    index = 0
```

Comment:
Can we try to use variable names which are more meaningful? For example current_number, next_number.